Gateway/Request/APMBuilder.php

```diff
@@ -9,13 +9,14 @@
 use Omise\Payment\Model\Config\Internetbanking;
 use Omise\Payment\Model\Config\Tesco;
 use Omise\Payment\Model\Config\Truemoney;
+use Omise\Payment\Model\Config\Conviniencestore;
 use Omise\Payment\Observer\InstallmentDataAssignObserver;
 use Omise\Payment\Observer\InternetbankingDataAssignObserver;
 use Omise\Payment\Observer\TruemoneyDataAssignObserver;
+use Omise\Payment\Observer\ConviniencestoreDataAssignObserver;
 
 class APMBuilder implements BuilderInterface
 {
-
     /**
      * @var string
```

Comment:
Should be @const ?

Reply:
hmm.. never saw, I think proper way is putting `var` in doc

Reply:
Magento 2 docs seem to suggest putting neither. Just a comment if necessary